### ISSUE TITLE
[Vue warn]: Invalid vnode type when creating vnode: undefined.

### ISSUE THREAD

**Issue by zhaozhongyu** [devilwjp/veaury#85]
helloworld.vue内容
```
<script setup>
  import { onMounted } from 'vue';
    onMounted(() => {
        console.log('mounted');
    })

</script>

<template>
  <div>hello world</div>
</template>
```

react中引用:
```
import { applyVueInReact, applyPureVueInReact } from 'veaury';
import helloworld from './helloworld.vue';

const HelloWorld = applyVueInReact(helloworld);

// <HelloWorld></HelloWorld>

```
此时控制台提示
![image](https://github.com/devilwjp/veaury/assets/8148957/8826846f-ebbb-4966-8e4f-f918fdce851b)

界面的元素:
![image](https://github.com/devilwjp/veaury/assets/8148957/2de46ada-1406-41d6-ac4b-8a3caac967a1)

**Comment by devilwjp** [comment]
@zhaozhongyu 提供一个范例项目给我

**Comment by Parag2385** [comment]
@zhaozhongyu Were you able to solve this issue?

**Comment by glepka** [comment]
I have the same issue
![image](https://github.com/gloriasoft/veaury/assets/105394707/0d1699fd-52fa-4a6a-b34a-437211155051)

**Comment by Parag2385** [comment]
@glepka , nope, I couldn't figure out. I ended up not using veaury. Instead, I'm developing the vue app screen by screen and using routing to run it by the side of our react app.

**Comment by devilwjp** [comment]
@Parag2385 @glepka Can anyone provide an example project?

**Comment by glepka** [comment]
> > > @Parag2385 @glepka Can anyone provide an example project?
> > 
> > 
> > what exactly needs to be sent?
> 
> Provide the github URL of a test project. @glepka


I created a private repository and added you as a collaborator 
https://github.com/glepka/vueinreact

There is a simple structure 
The Vue component should appear on the "/test" route, but this does not happen 


The vue component itself is located in the dist folder

**Comment by devilwjp** [comment]
> > > > @Parag2385 @glepka Can anyone provide an example project?
> > > 
> > > 
> > > what exactly needs to be sent?
> > 
> > 
> > Provide the github URL of a test project. @glepka
> 
> I created a private repository and added you as a collaborator https://github.com/glepka/vueinreact
> 
> There is a simple structure The Vue component should appear on the "/test" route, but this does not happen
> 
> The vue component itself is located in the dist folder

@glepka 
So is this project built using `create reat app`? It looks like you don't have webpack.config.js configured, I think you may need to run `npm run eject` first.
For details on how to configure webpack, please refer to https://github.com/gloriasoft/veaury/blob/master/dev-project-react/README.md

**Comment by devilwjp** [comment]
> > > > @Parag2385 @glepka Can anyone provide an example project?
> > > 
> > > 
> > > what exactly needs to be sent?
> > 
> > 
> > Provide the github URL of a test project. @glepka
> 
> I created a private repository and added you as a collaborator https://github.com/glepka/vueinreact
> 
> There is a simple structure The Vue component should appear on the "/test" route, but this does not happen
> 
> The vue component itself is located in the dist folder

@glepka 
So is this project built using `create reat app`? It looks like you don't have webpack.config.js configured, I think you may need to run `npm run eject` first.
For details on how to configure webpack, please refer to https://github.com/gloriasoft/veaury/tree/master/dev-project-react/README.md

**Comment by devilwjp** [comment]
@glepka Is the problem solved?
